CHANGELOG.rst

```diff
@@ -2,6 +2,171 @@ Changelog
 =========
 
 
+v0.52 (2020-04-23)
+------------------
+
+Fix
+~~~
+- Handling of dictionary-values in WebElement.get_attribute() (#521)
```

Comment:
It makes sense to mention that this is the last version supporting Python 2 (because of its EOL)